### PR TITLE
Added support for CollectSourceInformation flag

### PR DESCRIPTION
**Pull request by Shyam-Gupta**
Refer RFC: https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0010-Source-Information-For-Discovered-Tests.md
Added support for handling CollectSourceInformation flag in runsettings in NUnit VS adapter.

### CODE REVIEW

**Comment by OsirisTerje** [comment]
This need some explanation (and add an issue please).  I see the changes, but I am not able to understand what this is doing.

**Comment by codito** [comment]
Created issue https://github.com/nunit/nunit3-vs-adapter/issues/351 for this change.

File: src/NUnitTestAdapter/AdapterSettings.cs [hunk 1]
```diff
@@ -101,6 +103,7 @@ public void Load(string settingsXml)
             TargetPlatform = GetInnerText(runConfiguration, "TargetPlatform");
             TargetFrameworkVersion = GetInnerText(runConfiguration, "TargetFrameworkVersion");
             TestAdapterPaths = GetInnerText(runConfiguration, "TestAdapterPaths");
+            CollectSourceInformation = GetInnerTextAsBool(runConfiguration, "CollectSourceInformation", true);
```
Comment (codito) [inline src/NUnitTestAdapter/AdapterSettings.cs]: `CollectSourceInformation` is true by default to allow for older VS to work seamlessly, they will never set the setting.

**Comment by jnm2** [comment]
I like this.

**Pull request merged**